x/profile/client/rest/query.go

```diff
@@ -0,0 +1,47 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/desmos-labs/desmos/x/profile/internal/types"
+	"github.com/gorilla/mux"
+)
+
+func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/profile/{moniker}", queryProfileHandlerFn(cliCtx)).Methods("GET")
```

Comment:
I think it's better to have this endpoint more coherent with the generic one, changing it to
```suggestion
	r.HandleFunc("/profiles/{moniker}", queryProfileHandlerFn(cliCtx)).Methods("GET")
```